docs/user_guides/projects/jobs/schedule_job.md

```diff
@@ -6,7 +6,7 @@ description: Documentation on how to schedule a job on Hopsworks.
 
 ## Introduction
 
-Hopsworks jobs can be scheduled to run at regular intervals using the scheduling function provided by Hopsworks. Each job can be configured to have a single schedule.
+Hopsworks clusters can run jobs on a schedule, allowing you to automate the execution of your jobs. Whether you need to backfill your feature groups on a nightly basis or run a model training pipeline every week, the Hopsworks scheduler can help you automate these tasks. Each job can be configured to have a single schedule. For more complex use cases, Hopsworks integrates with the open-source [Apache Airflow](https://airflow.apache.org/use-cases/), check out our [Airflow Guide](../airflow/airflow.md).
```

Comment:
did few edits, but also, not sure what 
> Each job can be configured to have a single schedule
actually means... I am being pendantic here, but its maybe me just not being technical enough. 

```suggestion
Hopsworks clusters can run jobs on a schedule, allowing you to automate the execution. Whether you need to backfill your feature groups on a nightly basis or run a model training pipeline every week, the Hopsworks scheduler will help you automate these tasks. Each job can be configured to have a single schedule. For more advanced use cases, Hopsworks integrates with any DAG manager and directly with the open-source [Apache Airflow](https://airflow.apache.org/use-cases/), check out our [Airflow Guide](../airflow/airflow.md).
```